lib/miq_automation_engine/models/miq_ae_yaml_import.rb

```diff
@@ -8,6 +8,10 @@ def initialize(domain, options)
     @domain_name = domain
     @options     = options
     @restore     = @options.fetch('restore', false)
+    tenant_id    = @options.fetch('tenant_id', Tenant.root_tenant.try(:id))
```

Comment:
@mkanoor by default, this converts the root object to an id, and then looks it up again.
would you change so it will use the id to look up an object but otherwise default to the root object?

ping if you want help.  lets get this merged